pyzeebe/channel/oauth_channel.py

```diff
@@ -0,0 +1,122 @@
+from functools import partial
+from typing import Callable, Optional
+
+import grpc
+from grpc.aio._typing import ChannelArgumentType
+from oauthlib import oauth2
+from requests_oauthlib import OAuth2Session
+
+from pyzeebe.credentials.oauth import (
+    Oauth2ClientCredentialsMetadataPlugin,
+    OAuth2MetadataPlugin,
+)
+
+
+def create_oauth2_client_credentials_channel(
+    target: str,
+    client_id: str,
+    client_secret: str,
+    authorization_server: str,
+    scope: Optional[str] = None,
+    audience: Optional[str] = None,
+    expire_in: Optional[int] = None,
+    channel_options: Optional[ChannelArgumentType] = None,
```

Comment:
`channel_credentials` should be presented in this method for using self-signed certificates.

Reply:
I will add it.